packages/appeals-service-api/src/services/document-metadata.service.js

```diff
@@ -1,6 +1,17 @@
 const logger = require('../lib/logger');
 const { DocumentMetadataRepository } = require('../repositories/document-metadata-repository');
 const ApiError = require('../errors/apiError');
+const Joi = require('joi');
+
+/**
+ * @typedef {import('../repositories/document-metadata-repository.js').DocumentMetadata} DocumentMetadata
+ * @typedef {import('../repositories/document-metadata-repository').UpdateResult} UpdateResult
```

Comment:
how come one of these has `.js` and the other doesn't?

Reply:
makes no difference, just wasn't consistent